src/http.rs

```diff
@@ -338,6 +345,41 @@ pub fn get_http_scheme_from_uri(uri: &Uri) -> &'static str {
     })
 }
 
+/// Builds a [TraceLayer] configured for a HTTP server.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I think it might be useful to add at least a sentence here describing what we use this for.